doc/code_snippets/snippets/connectors/go/hello.go

```diff
@@ -0,0 +1,105 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/tarantool/go-tarantool/v2"
+	"time"
+)
+
+func main() {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+	defer cancel()
+	dialer := tarantool.NetDialer{
+		Address:  "127.0.0.1:3301",
+		User:     "sampleuser",
+		Password: "123456",
+	}
+	opts := tarantool.Opts{
+		Timeout: time.Second,
+	}
+
+	conn, err := tarantool.Connect(ctx, dialer, opts)
+	if err != nil {
+		fmt.Println("Connection refused:", err)
+		return
+	}
+
+	// Interacting with the database
+	// Insert data
+	tuple1, err := conn.Do(
+		tarantool.NewInsertRequest("bands").
+			Tuple([]interface{}{1, "Roxette", 1986}),
+	).Get()
+	tuple2, err := conn.Do(
+		tarantool.NewInsertRequest("bands").
+			Tuple([]interface{}{2, "Scorpions", 1965}),
+	).Get()
+	tuple3, err := conn.Do(
+		tarantool.NewInsertRequest("bands").
+			Tuple([]interface{}{3, "Ace of Base", 1987}),
+	).Get()
+	tuple4, err := conn.Do(
+		tarantool.NewInsertRequest("bands").
+			Tuple([]interface{}{4, "The Beatles", 1960}),
+	).Get()
```

Comment:
The connector allows to execute requests in an asynchronous manner. It is a preferred way to make requests with the connector. It would be great to hightlight it in the example. It could be done only for `Insert` requests:

```suggestion
        tuples := [][]interface{
            []interface{}{1, "Roxette", 1986},
            []interface{}{2, "Scorpions", 1965},
            []interface{}{3, "Ace of Base", 1987},
            []interface{}{4, "The Beatles", 1960},
        }
        futures := []*tarantool.Future
        for _, tuple := range tuples {
             request := tarantool.NewInsertRequest("bands").Tuple(tuple)
             futures = append(futures, conn.Do(request))
        }

        fmt.Println("Inserted tuples:")
        for _, future := range futures {
             // Result tuples will be printed in a random order, but it is fine due to the async nature.
             result, err := future.Get()
             if err != nil {
                 fmt.Println("Got an error:", err)
             } else {
                 fmt.Println(result)
             }
        }
```

Reply:
Updated the sample and description, please see the staging:

https://docs.d.tarantool.io/en/doc/connectors/how-to/getting_started_go/#inserting-data